coldfront/plugins/qumulo/utils/acl_allocations.py

```diff
@@ -177,15 +178,49 @@ def set_allocation_attributes(
             value=f"storage-{wustlkey}-{acl_type}",
         )
 
+    @staticmethod
+    def get_allocation_rwro_group_name(access_allocations, rwro):
+        allocation = next(
+            filter(
+                lambda access_allocation: access_allocation.resources.filter(
+                    name=rwro
+                ).exists(),
+                access_allocations,
+            )
+        )
+        return allocation.get_attribute(name="storage_acl_name")
+
+    @staticmethod
+    def reset_allocation_acls(allocation: Allocation, qumulo_api: QumuloAPI):
+        return AclAllocations.set_or_reset_allocation_acls(
+            allocation,
+            qumulo_api,
+            True
+        )
+
     @staticmethod
     def set_allocation_acls(
```

Comment:
This method really isn't doing anything.  I would just take this out and replace references of it with calls to `set_or_reset_allocation_acls`